docs/source/kale.embed.rst

```diff
@@ -30,7 +29,6 @@ kale.embed.gcn module
    :members:
    :undoc-members:
    :show-inheritance:
-   :exclude-members: message, update
```

Comment:
May I ask why the exclude-members are deleted? In my understanding the message and update should be private methods and not be emerged in docs. I follow the rules on pytorch-geometrics doc: https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.conv.GCNConv

Reply:
I have thought it was the cause of the missing docs. I proved that I was wrong. It was due to the libraries. I will reverse and test.